vectordraw.js

```diff
@@ -222,16 +264,31 @@ VectorDraw.prototype.removeVector = function(idx) {
     if (object) {
         this.board.removeAncestors(object);
         // Enable the <option> element corresponding to vector.
-        var option = this.element.find('.menu option[value=' + idx + ']');
+        var option = this.element.find('.menu option[value=vector-' + idx + ']');
         option.prop('disabled', false);
     }
 };
 
-VectorDraw.prototype.addVectorFromList = function() {
+VectorDraw.prototype.getSelectedElement = function() {
+    var selector = this.element.find('.menu select').val();
+    if (selector) {
+        selector = selector.split('-');
```

Comment:
@e-kolpakov You have a point here.  When we change this, we would have to stop using the `value` attribute of `option`, since it can only take string values, and that would make the code harder to read and more involved in other regards.  Now that I have introduced the `getMenuOption()` function, dealing with the strings is isolated in just two small functions, so I feel it's acceptable now.